huntsman/camera/pyro.py

```diff
@@ -200,6 +203,8 @@ def take_exposure(self,
                       filename=None,
                       dark=False,
                       blocking=False,
+                      ngas_push=False,
+                      filename_ngas=None,
```

Comment:
If you're going to pass these options in to the class constructor shouldn't you be storing them as instance variables to use as defaults for the later function calls?